### PR TITLE
add new subpage legacyConnect and fix signUpMobile sign__title and si…

### PR DESCRIPTION
**Pull request by Ervige**
…gn__line

### CODE REVIEW

File: pages/housePrideQuiz/legacyConnect.css [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    position: absolute;
+}
+
+.steps__squareOuter {
+    position: absolute;
+    bottom: -2px;
+    left: -6px;
+    border-left: 3px solid var(--squareBackground);
+    border-right: 3px solid var(--squareBackground);
+    border-top: 3px solid var(--squareBackground);
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/legacyConnect.css:143]: You implement 3 lines with the same code. Change it - style for border: 3px solid var(--squareBackground) and border-bottom with different style

File: pages/housePrideQuiz/legacyConnect.css [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    border-bottom: 28px solid var(--squareBackground);
+}
+
+.steps__squareOuter1 {
+    position: absolute;
+    bottom: -2px;
+    left: 11px;
+    border-left: 3px solid var(--squareBackground);
+    border-right: 3px solid var(--squareBackground);
+    border-top: 3px solid var(--squareBackground);
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/legacyConnect.css:154]: same topic here

File: pages/housePrideQuiz/legacyConnect.html [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+        </h4>
+        <a href="#" class="legacy__link">EXPLORE THE GAME</a>
+      </div>
+      <div class="rewards">
+        <h1 class="rewards__title">Unlock Exclusive Rewards</h1>
+        <h3 class="rewards__undertitle">follow these steps</h3>
+      </div>
+      <div class="steps">
+        <div class="steps__container">
+          <div class="steps__containerBox">
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/legacyConnect.html:84]: You have class "container" and "containerBox" - this is nonsense - each class must represent with its name what this tag is in the page structure. Reading this, I have no idea what role containerBox plays here. There is also too much nesting here - it shouldn't be like that.

File: pages/housePrideQuiz/legacyConnect.html [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+            <div class="steps__letter">
+              <h2 class="steps__title">step</h2>
+              <h1 class="steps__number">1</h1>
+            </div>
+            <h2 class="steps__title">...</h2>
+            <div class="steps__letter">
+              <h2 class="steps__title">step</h2>
+              <h1 class="steps__number">2</h1>
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/legacyConnect.html:92]: Do not use <h1> - h1 is ONLY used for main page titles or section names. Change it to e.g. H3 and set the size in the class.

File: pages/housePrideQuiz/legacyConnect.html [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+                and get the latest Wizarding World news and offers.
+              </h6>
+              <a
+                href="/pages/housePrideQuiz/signUpMobile.html"
+                class="steps__button"
+                >LOG IN</a
+              >
+              <div class="steps__square">
+                <div class="steps__squareOuter"></div>
+                <div class="steps__squareOuter1"></div>
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/legacyConnect.html:129]: What the..? What is _squareOuter1_ - change this class for normal name. BEM does not allow you to use class names with numbers. Before you write something - read and learn what BEM is, and how classnames should look like -> https://getbem.com/naming/

File: pages/housePrideQuiz/legacyConnect.html [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+        </div>
+      </div>
+      <div class="unlock">
+        <h1 class="unlock__title">
+          Unlock exclusive rewards when you link account
+        </h1>
+        <figure>
+          <img
+            src="https://www.wizardingworld.com/assets/_next/static/images/promo-sorting@2x-1d78011609faa33566d21de2d0a11457.png"
+            alt=""
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/legacyConnect.html:144]: Why alt attribute is empty?

**Review by AlpacaInTheCode — CHANGES_REQUESTED** [review]
Some places in this code looks like pushed to repo without quality checking before pull request.

File: pages/housePrideQuiz/legacyConnect.css (hunk 1 above)
Comment (Ervige) [reply pages/housePrideQuiz/legacyConnect.css:143]: fixed

File: pages/housePrideQuiz/legacyConnect.css (hunk 2 above)
Comment (Ervige) [reply pages/housePrideQuiz/legacyConnect.css:154]: fixed

File: pages/housePrideQuiz/legacyConnect.html (hunk 3 above)
Comment (Ervige) [reply pages/housePrideQuiz/legacyConnect.html:84]: fixed

File: pages/housePrideQuiz/legacyConnect.html (hunk 4 above)
Comment (Ervige) [reply pages/housePrideQuiz/legacyConnect.html:92]: fixed

File: pages/housePrideQuiz/legacyConnect.html (hunk 5 above)
Comment (Ervige) [reply pages/housePrideQuiz/legacyConnect.html:129]: fixed

File: pages/housePrideQuiz/legacyConnect.html (hunk 6 above)
Comment (Ervige) [reply pages/housePrideQuiz/legacyConnect.html:144]: fixed

**Review by AlpacaInTheCode — APPROVED** [review]

**Pull request merged**